src/Components/RecsListTable/index.js

```diff
@@ -53,6 +54,8 @@ const RecsListTable = () => {
   const page = filters.offset / filters.limit + 1;
   const [filteredRows, setFilteredRows] = useState([]);
   const [displayedRows, setDisplayedRows] = useState([]);
+  const [recommendations, setRecommendations] = useState(recs);
+  const [sortBy, setSortBy] = useState({});
```

Comment:
Please, keep all filtering/sorting parameters in the global redux store. In the end, you should access sorting parameters in the same way as `filters` with `useSelector`.

Reply:
I will do that as a part of the refactor as soon as the sorting works without bugs